### PR TITLE
ci-vst.sh: use -o progs

### PR DESCRIPTION
**Pull request by SkySkimmer**
This is closer to what we mean than reproducing the default target
without progs.

### CODE REVIEW

**Comment by ejgallego** [comment]
I am OK with that but it still relies on an "internal" name, so breakage is possible.

File: dev/ci/ci-vst.sh [hunk 1]
```diff
@@ -8,10 +8,6 @@ VST_CI_DIR="${CI_BUILD_DIR}/VST"
...
-# HACK: from the upstream makefile:
-#
-# default_target: _CoqProject version.vo msl veric floyd progs
-#
-# We have to omit progs as otherwise we timeout on Travis; once we
-# move to Gitlab we will able to just use `make`
-( cd "${VST_CI_DIR}" && make IGNORECOQVERSION=true _CoqProject version.vo msl veric floyd )
+# We have to omit progs as otherwise we timeout on Travis; on Gitlab
+# we will able to just use `make`
+( cd "${VST_CI_DIR}" && make IGNORECOQVERSION=true -o progs )
```
Comment (ejgallego) [inline dev/ci/ci-vst.sh]: Please improve/keep the old documentation, for the casual reader it may be clear what is going on.

Comment (SkySkimmer) [reply dev/ci/ci-vst.sh]: It says "we omit progs", that's as clear as it gets.

Comment (ejgallego) [reply dev/ci/ci-vst.sh]: Something such as:
```
The VST default is as of today:
default_target: ....
Due to build time constraints on Travis we omit the `progs` target, etc...
```

**Comment by SkySkimmer** [comment]
I think that's not a useful addition.


Gaëtan Gilbert

File: dev/ci/ci-vst.sh (hunk 1 above)
Comment (Zimmi48) [reply dev/ci/ci-vst.sh]: I think the comment is clear enough like this. However, it contains a typo: "we will *be* able to"...

**Comment by ejgallego** [comment]
I'd prefer the documentation to explain the issue in full detail.

**Comment by SkySkimmer** [comment]
It is in full detail.

Gaëtan Gilbert

**Comment by ejgallego** [comment]
No, it is not satisfactory to me. Even after reading that I'd have to look at the VST makefile to understand what is going on, so I'd like to see more verbose documentation [that never hurt anyone BTW].

**Comment by ejgallego** [comment]
So you don't plan to amend this?

**Comment by ejgallego** [comment]
> So you don't plan to amend this?

:/

**Comment by SkySkimmer** [comment]
I was going to but procrastination is too strong.

Gaëtan Gilbert

**Pull request merged**